dataactcore/scripts/load_federal_hierarchy.py

```diff
@@ -143,7 +143,9 @@ def pull_offices(sess, filename, update_db, pull_all, updated_date_from, export_
                         if not org.get('aacofficecode') or not org.get('agencycode') or not agency_code:
                             # Item from Fed Hierarchy is missing necessary data, ignore it
                             continue
-
+                        # store all the cgacs/subtiers loaded in from this run, to be filtered later
+                        metrics['missing_cgacs'].append(agency_code)
+                        metrics['missing_subtier_codes'].append(org.get('agencycode'))
```

Comment:
Does this only happen to the lower tier agencies? Do we want to include the `top_sub_levels` in case they don't have any offices listed under them?

Reply:
I would say keep out the `top_sub_levels` until we start processing them which will just remove lines 132-134.